tests/foreman/ui/test_myaccount.py

```diff
@@ -58,7 +57,8 @@ def setUp(self):
         # Creating user for each test to not mess with default user
         user = User(login=gen_alpha(), password='password')
         # copying missing id because create returns only password's hash once
-        # it is not stored in plain text for security reason
+        # it is not stored in plain text for security reason and original
```

Comment:
this apply also to some tests that can create users, the user creation is difficult to understand, do we really need this approach?